types/lib/sequelize.d.ts

```diff
@@ -166,7 +166,7 @@ export interface Config {
   };
 }
 
-export type Dialect = 'mysql' | 'postgres' | 'sqlite' | 'mariadb' | 'mssql';
+export type Dialect = 'mysql' | 'postgres' | 'sqlite' | 'mariadb' | 'mssql' | Object;
```

Comment:
Maybe we should create a type to represent an abstract dialect (https://github.com/sequelize/sequelize/blob/main/lib/dialects/abstract/index.js)

What do you think?

```suggestion
interface AbstractDialect {
  new (...args: any[]): any
}

export type Dialect = 'mysql' | 'postgres' | 'sqlite' | 'mariadb' | 'mssql' | AbstractDialect;
```

Reply:
good idea, I will create one.